scss/_tables.scss

```diff
@@ -48,6 +48,14 @@
   > tbody {
     vertical-align: inherit;
   }
+
+  // Boosted mod
+  // Add negative right margin for selectable rows
+  td:first-child .form-check,
+  th:first-child .form-check {
+    margin-right: -.75rem;
```

Comment:
Is there any reason for this specific value ? IMO, a greater value would be better here ?

Reply:
It seems to me that form-check is needed to surround checkbox input with form-check-input. 
Moreover, removing form-check does not solve the problem and removing padding is not ok (it is a left padding and checbox image is added as a background).

Reply:
For the value, it has been proposed by Isabelle but maybe you're right, -1rem could be nicer...

With -0.75rem :
![image](https://user-images.githubusercontent.com/16681352/170003777-bfb2dfec-d58d-484a-b2ac-2194341fba64.png)

With -1rem :
![image](https://user-images.githubusercontent.com/16681352/170004859-7a725b7a-f512-4614-9a18-706fb80562fe.png)

Reply:
Yes the 1rem looks better regarding the specs ! If I were a bit picky, I'd say that `1.0625rem` would be pixel perfect and IMO, `$spacer * .85` would be even more understandable.
The management of checkboxes seems to be a bit weird to me even on Bs side, wondering if there is anything to change on their side ... I don't understand from where the spacing after the checkboxes come. The solution here looks fine to me